sklearn/ensemble/base.py

```diff
@@ -49,8 +49,7 @@ def __init__(self, base_estimator, n_estimators=10,
 
         # Don't instantiate estimators now! Parameters of base_estimator might
         # still change. Eg., when grid-searching with the nested object syntax.
-        # This needs to be filled by the derived classes.
-        self.estimators_ = []
+        # self.estimators_ needs to be filled by the derived classes in fit.
```

Comment:
This is a slight backward incompatible change. I think that we should indeed just go ahead and do it. Do people (eg @ogrisel and @amueller ) agree?